### ISSUE TITLE
Request a osx-arm64 release on Conda

### ISSUE THREAD

**Issue by DavidT3** [PyAbel/PyAbel#384]
Hi!

First of all, thanks for making this package available - it is very useful in my own software. I am currently re-building the dependency list and attempting to make a cross-platform conda environment lock file, but currently you don't have a conda release specifically for osx-arm64, so I am unable to support one of the key platforms I need too. I can get around this by having users install pyabel later through pip (and that is what I will do), but if at some point you could add a release for osx-arm64 I would be very grateful.

**Comment by MikhailRyazanov** [comment]
I suspect that first we need to release a new version fully compatible with NumPy 2.0.

But regarding osx-arm64 for conda, do you have instructions how to add osx-arm64 to our https://github.com/conda-forge/pyabel-feedstock/? (I don't see any platforms listed there explicitly, only a lot of automatically generated stuff.)

**Comment by DanHickstein** [comment]
@DavidT3: we definitely want to support your request! But, we are a bunch of scientists that are only moderately skilled at Python, and fairly unskilled at releasing software. We appreciate your advice :)

**Comment by DavidT3** [comment]
Yeah I'm happy to help if I can! I'll take a look to see how releasing for arm64 on Conda works, I've still never actually released anything on Conda, but I've been meaning too so it'll be useful for me as well.

Well as another scientist fairly unskilled at releasing software, I'm grateful for your efforts!

**Comment by MikhailRyazanov** [comment]
The only thing I've found is https://conda-forge.org/blog/2020/10/29/macos-arm64/, and according to [How to add a osx-arm64 build to a feedstock](https://conda-forge.org/blog/2020/10/29/macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock) there:
> If you would like to add support, please send a PR adding the feedstock name to the above list

which is [osx_arm64.txt](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/osx_arm64.txt) at [conda-forge / conda-forge-pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock). So maybe it will be that simple...

**Comment by MikhailRyazanov** [comment]
@DavidT3, I was able to _build_ osx-arm64 conda packages for PyAbel 0.9.1. However, they are _not tested at all_. So please try running all the tests you can and report back, so that hopefully we can close this issue (after over a year...).

PyAbel 0.9.0 will not be rebuilt because it doesn't really work with current versions of other conda packages.
